TwitchLib.Client.Models/Builders/OutboundChatMessageBuilder.cs

```diff
@@ -1,10 +1,12 @@
-﻿namespace TwitchLib.Client.Models.Builders
+﻿#nullable disable
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace TwitchLib.Client.Models.Builders
 {
     public sealed class OutboundChatMessageBuilder : IBuilder<OutboundChatMessage>
     {
         private string _channel;
         private string _message;
-        private string _userName;
```

Comment:
so i know the username here seemed kind of redundant, but i believe it was added because there were users of the library that had a single OutboundChatMessage event for multiple bot instances using multiple users, so they wanted to know which account/bot was sending the outgoing message.

Not apposed to removing though.